nltk/compat.py

```diff
@@ -121,6 +126,131 @@ def load_module(self, name):
 
     sys.meta_path.insert(0, TkinterLoader())
 
+    import pytz
```

Comment:
I think that pytz dependency is fine, it is a commonly used package. Though it is used only for pytz.utc; there is an example UTC tzinfo implementation in Python 2.x docs: https://docs.python.org/2/library/datetime.html#tzinfo-objects 

```
from datetime import tzinfo, timedelta, datetime

ZERO = timedelta(0)
HOUR = timedelta(hours=1)

# A UTC class.

class UTC(tzinfo):
    """UTC"""

    def utcoffset(self, dt):
        return ZERO

    def tzname(self, dt):
        return "UTC"

    def dst(self, dt):
        return ZERO

utc = UTC()
```
It may be short enough to bundle, without adding a dependency.